factory/tools/analyze_entries

```diff
@@ -564,8 +564,7 @@ Glidein log analysis for frontend %s - %s
     ######################################################################
 
     # sort periods from least to greatest, with 24 hours at the top
-    period_list = period_data.keys()
-    period_list.sort()
+    period_list = sorted(period_data.keys())
```

Comment:
Should it be explicitly converted to list?
`period_list = sorted(list(period_data.keys()))`
or is sorted taking care of it?

Reply:
Sorted takes care of it